docs/rst/recording/usage/configuration.rst

```diff
@@ -339,10 +339,46 @@ The recorder output file does support the following configuration settings under
         - ``unsigned int``
         - ``0``
 
+    *   - Resource limits
+        - ``resource-limits``
+        - :ref:`recorder_usage_configuration_resource_limits`
+        - ``list``
+        - ``unlimited``
+
 When DDS Recorder application is launched (or when remotely controlled, every time a ``start/pause`` command is received while in ``SUSPENDED/STOPPED`` state), a temporary file with ``filename`` name (+timestamp prefix) and ``.mcap.tmp~`` extension is created in ``path``.
 This file is not readable until the application is terminated (or a ``suspend/stop/close`` command is received).
 On such event, the temporal file is renamed to have ``.mcap`` extension in the same location, and is then ready to be processed.
 
+.. _recorder_usage_configuration_resource_limits:
+
+Resource Limits
+"""""""""""""""
+
+The ``resource-limits`` tag allows users to limit the size of the *DDS Recorder's* output.
+The ``max-file-size`` tag specifies the maximum size of each output file and the ``max-size`` tag specifies the maximum aggregate size of all output files.
```

Comment:
Related to https://github.com/eProsima/DDS-Record-Replay/pull/129#discussion_r1553435350

Reply:
There's a note to explain the case you're mentioning. Is that enough?

Reply:
Not really in my opinion, but doesn't matter as long as it gets fixed in the upcoming PR.

Reply:
I could try to fix it if you explained what you meant with:

> Not totally true according to my interpretation given the current implementation.

If not I guess it can wait.